### PR TITLE
feat(tools): add `get_updated_now()` in time tools

### PR DESCRIPTION
**Pull request by chronolaw**
### Summary

KAG-5280

### Checklist

- [ ] The Pull Request has tests
- [ ] A changelog file has been created under `changelog/unreleased/kong` or `skip-changelog` label added on PR if changelog is unnecessary. [README.md](https://github.com/Kong/gateway-changelog/blob/main/README.md)
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Issue reference

Fix #_[issue number]_

### CODE REVIEW

**Review by bungle — COMMENTED** [review]
It looks like this is also quite common: aka `get_duration(start)`

File: kong/clustering/control_plane.lua [hunk 1]
```diff
@@ -173,8 +173,7 @@ function _M:push_config()
     n = n + 1
   end
 
-  ngx_update_time()
-  local duration = ngx_now() - start
+  local duration = now_updated() - start
```
Comment (bungle) [inline kong/clustering/control_plane.lua:176]: I mean this.

Comment (chronolaw) [reply kong/clustering/control_plane.lua:176]: I searched the repo by `git grep 'now_updated() -'`, only found two places, perhaps it is not the time to add this function.

Comment (bungle) [reply kong/clustering/control_plane.lua:176]: `- start` reveals a lot more. And there may be many more too.

Comment (bungle) [reply kong/clustering/control_plane.lua:176]: but we can do it in other pr for sure, if we decide to do it.

Comment (chronolaw) [reply kong/clustering/control_plane.lua:176]: Yes, Let's leave it to another PR, keep this PR focus on one topic.

**Review by Water-Melon — APPROVED** [review]

**Review by bungle — APPROVED** [review]

**Pull request merged**